src/frequenz/sdk/actor/_power_managing/_base_classes.py

```diff
@@ -18,9 +19,35 @@ class Proposal:
 
     source_id: str
     preferred_power: Power
-    bounds: tuple[Power, Power]
+    bounds: tuple[Power, Power] | None
     battery_ids: frozenset[int]
     priority: int
+    request_timeout: datetime.timedelta = datetime.timedelta(seconds=5.0)
+    """The maximum amount of time to wait for the request to be fulfilled."""
+
+    adjust_power: bool = True
+    """Whether to adjust the power to match the bounds.
+
+    If `True`, the power will be adjusted (lowered) to match the bounds, so
+    only the reduced power will be set.
+
+    If `False` and the power is outside the batteries' bounds, the request will
+    fail and be replied to with an `OutOfBound` result.
+    """
+
+    include_broken_batteries: bool = False
```

Comment:
Did we want to remove all broken battery related code?

Reply:
Yes, didn't want to hide it here, because then we might not have the motivation to remove it from the PowerDistributor later.  I'll pick that up after this PR.